axon-framework/events/event-bus-and-event-store.md

```diff
@@ -393,14 +393,30 @@ If no explicit `eventSerializer` is configured, events are serialized using the
 
 ## Distributing Events
 
-The use of an `EventStore` provides a simple way of distributing events: by sharing the data source.
-This is the approach taken by [Axon Server](../../axon-server-introduction.md), for example.
+To distribute events between applications, it is paramount to know whether the applications belong to the same [bounded context](../../architecture-overview/ddd-cqrs-concepts.md#bounded-context).
+If they do, they're allowed to understand each other's language.
+In other words, they should be free to understand the events altogether.
 
-Such an approach will also suffice whenever the [`EmbeddedEventStore`](#embedded-event-store) is used.
-Doing so requires the correct configuration of each application instance to point to the same data source.
-Listing _all_ possible data sources is out of the scope of this guide however.
-The documentation for the chosen source should thus be read for specifics on this.
+As such, we can share the `EventStore`'s data source between these applications typically suffices.
```

Comment:
Well, grammarly is the tool, not the rule. The sentence doesn't make sense as it is right now.
Grammarly probably suggests "we can share" over "sharing" because it want to avoid the passive tense. However, we should remove the "typically suffices" from the sentense.